sklearn/preprocessing/tests/test_label.py

```diff
@@ -350,8 +352,11 @@ def test_sparse_output_multilabel_binarizer():
             assert_array_equal([1, 2, 3], mlb.classes_)
             assert mlb.inverse_transform(got) == inverse
 
-    with pytest.raises(ValueError):
-        mlb.inverse_transform(csr_matrix(np.array([[0, 1, 1], [2, 0, 0], [1, 1, 0]])))
+    for csr_container in CSR_CONTAINERS:
```

Comment:
Similarly you might also be able to move this into a separate test and parametrize that since this part is just checking for an error and is independent from the rest of the test.